src/main/java/crud/dao/impl/ManufacturerDaoJdBcImpl.java

```diff
@@ -0,0 +1,121 @@
+package crud.dao.impl;
+
+import crud.dao.abstraction.ManufacturerDao;
+import crud.lib.DaoImpl;
+import crud.lib.DataBaseException;
+import crud.model.Manufacturer;
+import crud.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@DaoImpl
+public class ManufacturerDaoJdBcImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO manufacturers (name,country) VALUES (?,?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement preparedStatement
+                    = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
```

Comment:
```suggestion
        try (Connection connection = ConnectionUtil.getConnection();
            PreparedStatement preparedStatement
                    = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
```
and then we can remove `statement.close()`